library/entropy_poll.c

```diff
@@ -50,26 +50,45 @@
 
 #include <windows.h>
 #if _WIN32_WINNT >= 0x0501 /* _WIN32_WINNT_WINXP */
-#include <wincrypt.h>
+#include <bcrypt.h>
+#if defined(_MSC_VER) && _MSC_VER <= 1600
```

Comment:
We don't need any of the pragma bits - we support MSVS 2013, which means `_MSC_VER` is always >= 1800

Reply:
Removed